coral/src/services/type-utils.ts

```diff
@@ -2,14 +2,11 @@ function objectHasProperty<T extends string>(
   object: unknown,
   key: T
 ): object is Record<T, unknown> {
-  if (
+  return !!(
```

Comment:
Do we need the type coercion? Isn't this achieving the same result?
```ts
function objectHasProperty<T extends string>(
  object: unknown,
  key: T
): object is Record<T, unknown> {
  return (
    object !== null &&
    object !== undefined &&
    Object.prototype.hasOwnProperty.call(object, key)
  );
}
```

Reply:
True! Will change that, too.

Reply:
https://github.com/Aiven-Open/klaw/pull/1664/commits/ad39a98b4c8f23417964cde2c0382b3cba553e15